pkg/tests/api/alerting/api_alertmanager_test.go

```diff
@@ -2650,25 +2650,20 @@ func TestEval(t *testing.T) {
 
 // createFolder creates a folder for storing our alerts under. Grafana uses folders as a replacement for alert namespaces to match its permission model.
 // We use the dashboard command using IsFolder = true to tell it's a folder, it takes the dashboard as the name of the folder.
-func createFolder(t *testing.T, store *sqlstore.SQLStore, folderID int64, folderName string) (string, error) {
+func createFolder(t *testing.T, folderUID, grafanaListedAddr, login, password string) error {
```

Comment:
Note: calling the dashboard store directly does not set up the right permissions, so I've changed it to call the endpoint and use the full folder creation flow.